tokyomovie/tokyomovie
# Feat/improve sign up

This PR improves sign up in various ways:
- when you enter in your info on user admin, the form values persist even on error
- You can choose to reset a persons password
  - It will email the user with a link to reset their password
  - Only they may see that reset page 

In order to accomplish this, I had to add some stuff
- jwt stuff
- email stuff

In order to test this, you will need to 
- make a sendgrid account
- verify an email account on sendgrid
- make an api token
- setup your env vars with the verified email address as ADMIN_EMAIL_ADDRESS and api token in SENDGRID_API_TOKEN
